sdcm/cluster.py

```diff
@@ -350,6 +350,7 @@ class BaseNode(AutoSshContainerMixin, WebDriverContainerMixin):  # pylint: disab
     CQL_PORT = 9042
     MANAGER_AGENT_PORT = 10001
     MANAGER_SERVER_PORT = 56080
+    max_coredump_thread_exceptions: int = 10
```

Comment:
Please, add one empty line before to separate this block.

Reply:
done